assets/app/styles/_list.less

```diff
@@ -0,0 +1,22 @@
+.confirm-delete {
```

Comment:
@jwforres thoughts on the location of these styles, or how deletion might factor into other list views?